contracts/Insurance.sol

```diff
@@ -199,7 +224,7 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
 
         uint256 ratio =
             PRBMathUD60x18.div(
-                getPoolTarget() - collateralAmount,
+                getPoolTarget() - collateralAmount - bufferAmount,
                 levNotionalValue
```

Comment:
If `levNotionalValue == 0`, this will throw. This can happen naturally, or happen when `maxlev == 1`

Reply:
@BenjaminPatch @kumar-ish

Reply:
There's a check for `levNotionalValue` on line 224; following block of code:
```
    if (levNotionalValue <= 0) {
            return 0;
        }
```